package.json

```diff
@@ -31,8 +33,11 @@
 		"test": "jest --verbose --coverage"
 	},
 	"dependencies": {
+		"@microsoft/fast-components": "^2.16.3",
```

Comment:
You can uninstall the `@microsoft/fast-components` package.

It's only used to get `provideFASTDesignsystem` right now (which I've mentioned in other comments can be replaced with `provideVSCodeDesignsystem`).

Reply:
Resolved by 82e710a8991eebce74fdb702c03f81171c6b684d